yocto/json2junit.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python3
+#
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
```

Comment:
Include usage

Reply:
Sorry Fisnik, could you clarify what you mean?

Reply:
If you check another script as an example, such as [smoke-test script](https://github.com/Pelagicore/vagrant-cookbook/blob/master/yocto/runqemu-smoke-test.sh), we have there a line that describes how to call the script. 

# Usage: smoke-test-images.sh <yoctodir> <images>

I am looking for something similar.

Reply:
Ah, silly me! Sure, I'll add it :-)